### PR TITLE
[WIP]: checkin install script for git-secrets

### PR DESCRIPTION
**Pull request by mountain-nerd**

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -81,15 +81,28 @@ function install_tilt() {
...
+    echo -e "${red}git-secrets installer is not supported for your platform: ${OS} ${reset}"
+    echo -e "${red}Please file an issue at https://github.com/awslabs/git-secrets/issues/new ${reset}"
+    exit 1
+  fi
+}
+
 function install() {
   VEGA_PATH=$(command -v vega 2>&1 || true)
   TILT_PATH=$(command -v tilt 2>&1 || true)
+  GIT_SECRETS_PATH=$(command -v tilt 2>&1 || true)
```
Comment (vs4vijay) [inline scripts/install.sh]: I think `tilt` should be `git-secret` in `GIT_SECRETS_PATH`

File: pkg/git_secrets/git_secrets.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+	"path/filepath"
+)
+
+const (
+	commandName  = "git-secrets"
+	RequiredText = `
+		git-secrets is not installed, which is required to run the application. 
+	`
+	InstallInstructions = `
+		Install using: curl -fsSL https://raw.githubusercontent.com/srijanone/vega/develop/scripts/install.sh| bash
```
Comment (vs4vijay) [inline pkg/git_secrets/git_secrets.go]: This script url should be install.sh or setup_git_secrets.sh

File: scripts/install.sh (hunk 1 above)
Comment (mountain-nerd) [reply scripts/install.sh]: :+1:

File: pkg/git_secrets/git_secrets.go (hunk 2 above)
Comment (mountain-nerd) [reply pkg/git_secrets/git_secrets.go]: Yes, makes sense to decouple this.

**Pull request merged**